unity/Assets/Scripts/Content/ContentData.cs

```diff
@@ -1423,20 +1540,29 @@ public GenericData(string name_ini, Dictionary<string, string> content, string p
 
         // If image specified it is relative to the path of the ini file
         // absolute paths are not supported
-        if (content.ContainsKey("image"))
```

Comment:
This change doesn't appear to be used (isn't tied to pps and images have extensions removed).

Reply:
This is to have alternative images from

```
[TokenInteract]
image="{import}/img/SpriteAtlasTexture-ATLAS_FEATURES_CORE (Group 0)-2048x1024-fmt12"
image2="{import}/img/SpriteAtlasTexture-ATLAS_FEATURES_CORE (Group 0)-2048x1024-fmt47"
x=1018
y=0
height=130
width=130
```

Reply:
Ah OK.